CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 ## Unreleased
```

Comment:
Could you change this to `## v0.7.2 (2025-01-29)` - so that we don't have to update in another PR?

Also, we could update crate versions to v0.7.2 in this PR as well.